node/cli/src/chain_spec.rs

```diff
@@ -185,25 +244,23 @@ pub fn darwinia_genesis(
 ) -> GenesisConfig {
 	let endowed_accounts: Vec<AccountId> = endowed_accounts.unwrap_or_else(|| {
```

Comment:
If the `endowed_accounts` is `None`, this will give a testnet default accounts set.

Okay, move the origin code to `testnet_genesis`. And Keep the `darwinia_genesis`.

Only the `gen` function use `darwinia_genesis`. Other testnet chains run with `testnet_genesis`.